src/scipp/core/variable.py

```diff
@@ -317,14 +317,34 @@ def vectors(*,
 
 
 def array(*,
-          dims: _Iterable,
-          values: array_like,
+          dims: _Optional[_Iterable] = None,
+          values: _Optional[array_like] = None,
           variances: _Optional[array_like] = None,
           unit: _Union[_cpp.Unit, str] = _cpp.units.dimensionless,
-          dtype: type(_cpp.dtype.float64) = None) -> _cpp.Variable:
+          dtype: type(_cpp.dtype.float64) = None,
+          **kwargs) -> _cpp.Variable:
     """Constructs a :class:`Variable` with given dimensions, containing given
     values and optional variances. Dimension and value shape must match.
-    Only keyword arguments accepted.
+
+    An optional arbitrary keyword argument is supported as convenience for creating
```

Comment:
I think it makes sense to show an example of the general case first, before talking about `kwargs`.